### PR TITLE
Update account-dashboard-requisition-lists.md

### PR DESCRIPTION
**Pull request by sudhanshu-bajaj**
Requisition List cannot be added from shopping cart.

## Purpose of this pull request

This pull request (PR) ...

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/v2.3/customers/account-dashboard-requisition-lists.html
- https://docs.magento.com/user-guide/customers/account-dashboard-requisition-lists.html

## Links to Magento source code or PRs

- ...
- ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/customers/account-dashboard-requisition-lists.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ _Requisition Lists_
 
 ## Add products to a requisition list
 
-Products can be added to a requisition list from product pages in the storefront, from the shopping cart, and from existing orders.
+Products can be added to a requisition list from product pages in the storefront, and from existing orders.
```
Comment (shrielenee) [inline src/customers/account-dashboard-requisition-lists.md:36]:
```suggestion
Products can be added to a requisition list from product pages in the storefront and from existing orders.
```

**Review by shrielenee — APPROVED** [review]
Editorial review only; technical review is still needed. Approved with grammar fix.

**Comment by leslietilling** [comment]
running tests

**Pull request merged**